apis/v1alpha1/forwarding_target_types.go

```diff
@@ -79,12 +80,37 @@ type HTTPForwardToTarget struct {
 	Filters []HTTPRouteFilter `json:"filters"`
```

Comment:
Can we start with an arbitrary limit of 8 or 16 filters?
We can certainly bump it up as needed in future. The reasoning here is that this limit acts as a warning or alert to us that this feature is being used a lot and in certain ways that we might not have anticipated and it might be helpful to take action --- let the community be creative or course correct.

Reply:
Set to 16 in both cases as a starting point. Seems pretty generous.